src/main/java/frc/robot/commands/RunIntake.java

```diff
@@ -22,6 +22,6 @@ public void initialize() {
     }
     @Override
     public void end(boolean interrupted) {
-        intake.run(0);
+        intake.getBeamBreak().beamBroken();
```

Comment:
this code will run when the command ends so it needs to stop the motor. The check of the beam break should happen in isFinished